test/enrollment_test.rb

```diff
@@ -3,49 +3,40 @@
 require "enrollment"
 
 class EnrollmentTest < Minitest::Test
-  def test_we_can_create_an_instance
-    enrollment = Enrollment.new(name: "ACADEMY 20",
-                                kindergarten_participation: {
+  def setup
```

Comment:
Duplicated setup code in some tests below was removed and this setup method was created. Good show on the isolated nature of these tests.